### PR TITLE
Fix whats this scroll top

### PR DESCRIPTION
**Pull request by vinidziuba**
### Description (*)

Solution : changed the tag for the strong tag and took the href.
### Fixed Issues (if relevant)

1. magento/magento2#24511: Click "What 's this" link scroll to the top of page

### Manual testing scenarios (*)

1. Go to backend
 Store->Configuration->Customer->Persistent Shopping Cart->Enable Persistence to Yes.
2. Go to Frontend, Sign In
3. In Remember me, click "What's this?" link

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by kalpmehta — CHANGES_REQUESTED** [review]
@vinidziuba Please fix all the space issues in the PR

File: app/code/Magento/Persistent/view/frontend/templates/remember_me.phtml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+
```
Comment (kalpmehta) [inline app/code/Magento/Persistent/view/frontend/templates/remember_me.phtml:2]: remove the empty line

File: app/code/Magento/Persistent/view/frontend/templates/remember_me.phtml [hunk 2]
```diff
@@ -13,10 +14,12 @@
 ?>
 <div id="remember-me-box" class="field choice persistent">
     <?php $rememberMeId = 'remember_me' . $block->getRandomString(10); ?>
-    <input type="checkbox" name="persistent_remember_me" class="checkbox" id="<?= $block->escapeHtmlAttr($rememberMeId) ?>"<?php if ($block->isRememberMeChecked()) : ?> checked="checked"<?php endif; ?> title="<?= $block->escapeHtmlAttr(__('Remember Me')) ?>" />
+    <input type="checkbox" name="persistent_remember_me" class="checkbox" id="<?= $block->escapeHtmlAttr($rememberMeId) ?>" <?php if ($block->isRememberMeChecked()) : ?> checked="checked" <?php endif; ?> title="<?= $block->escapeHtmlAttr(__('Remember Me')) ?>" />
```
Comment (kalpmehta) [inline app/code/Magento/Persistent/view/frontend/templates/remember_me.phtml:17]: remove the space you added here

File: app/code/Magento/Persistent/view/frontend/templates/remember_me.phtml [hunk 3]
```diff
@@ -13,10 +14,12 @@
...
     <label for="<?= $block->escapeHtmlAttr($rememberMeId) ?>" class="label"><span><?= $block->escapeHtml(__('Remember Me')) ?></span></label>
     <span class="tooltip wrapper">
-        <a class="link tooltip toggle" href="#"><?= $block->escapeHtml(__('What\'s this?')) ?></a>
+        <strong class="tooltip toggle">
+            <?= $block->escapeHtml(__('What\'s this?')) ?>
```
Comment (kalpmehta) [inline app/code/Magento/Persistent/view/frontend/templates/remember_me.phtml]: make this in one line like original code

File: app/code/Magento/Persistent/view/frontend/web/template/remember-me.html [hunk 4]
```diff
@@ -6,11 +6,13 @@
 -->
 <!-- ko if: isRememberMeCheckboxVisible() -->
 <div id="remember-me-box" class="field choice persistent">
-    <input type="checkbox" name="persistent_remember_me" class="checkbox" id="persistent_remember_me" data-bind="checked: isRememberMeCheckboxChecked, attr: {title: $t('Remember Me'), 'data-scope': dataScope}" />
+    <input type="checkbox" name="persistent_remember_me" class="checkbox" id="persistent_remember_me"
+        data-bind="checked: isRememberMeCheckboxChecked, attr: {title: $t('Remember Me'), 'data-scope': dataScope}" />
```
Comment (kalpmehta) [inline app/code/Magento/Persistent/view/frontend/web/template/remember-me.html]: in one line please

File: app/code/Magento/Persistent/view/frontend/web/template/remember-me.html [hunk 5]
```diff
@@ -6,11 +6,13 @@
...
     <label for="persistent_remember_me" class="label"><span data-bind="i18n: 'Remember Me'"></span></label>
     <span class="tooltip wrapper">
-        <a class="link tooltip toggle" href="#" data-bind="i18n: 'What\'s this?'"></a>
-        <span class="tooltip content" data-bind="i18n: 'Check \'Remember Me\' to access your shopping cart on this computer even if you are not signed in.'"></span>
+        <strong class="tooltip toggle" data-bind="i18n: 'What\'s this?'"></strong>
+        <span class="tooltip content"
+            data-bind="i18n: 'Check \'Remember Me\' to access your shopping cart on this computer even if you are not signed in.'"></span>
```
Comment (kalpmehta) [inline app/code/Magento/Persistent/view/frontend/web/template/remember-me.html]: in one line please

**Review by kalpmehta — CHANGES_REQUESTED** [review]

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=24602) <br/>All committers have signed the CLA.

**Review by vinidziuba — COMMENTED** [review]
Okay

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-5935](https://jira.corp.magento.com/browse/ENGCOM-5935) has been created to process this Pull Request
:eight_spoked_asterisk: @sidolov, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by magento-engcom-team** [comment]
@vinidziuba thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Review by kalpmehta — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @kalpmehta, thank you for the review.
[ENGCOM-5935](https://jira.corp.magento.com/browse/ENGCOM-5935) has been created to process this Pull Request
:eight_spoked_asterisk: @kalpmehta, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Pull request merged**